modules/tinymce/src/plugins/lists/test/ts/browser/ApplyTest.ts

```diff
@@ -900,6 +900,17 @@ describe('browser.tinymce.plugins.lists.ApplyTest', () => {
     TinyAssertions.assertRawContent(editor, expected);
   });
 
+  it('TINY-10136: Do not list-convert too many elements', () => {
```

Comment:
Could we describe the scenario a bit better here. Not sure of the exact wording but it basically a problem it it trying to move the start and end container position outside of the block if it's applied to an empty inline element inside that block.

Reply:
Sure, I was trying to keep it short, but might have been too short.